tools/StaticAnalysis/Exceptions/AzureRM.Automation/SignatureIssues.csv

```diff
@@ -59,3 +59,42 @@
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8420","Parameter set 'Windows' of cmdlet 'New-AzureRmAutomationSoftwareUpdateConfiguration' contains at least one parameter with a position larger than four, which is discouraged.","Limit the number of positional parameters in a single parameter set to four or fewer."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8420","Parameter set 'Linux' of cmdlet 'New-AzureRmAutomationSoftwareUpdateConfiguration' contains at least one parameter with a position larger than four, which is discouraged.","Limit the number of positional parameters in a single parameter set to four or fewer."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.NewAzureAutomationSoftwareUpdateConfiguration","New-AzureRmAutomationSoftwareUpdateConfiguration","1","8510","Cmdlet 'New-AzureRmAutomationSoftwareUpdateConfiguration' has multiple parameter sets, but no defined default parameter set.","Define a default parameter set in the cmdlet attribute."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement.RemoveAzureAutomationSoftwareUpdateConfiguration","Remove-AzureRmAutomationSoftwareUpdateConfiguration","1","8600","Cmdlet 'Remove-AzureRmAutomationSoftwareUpdateConfiguration' has no defined output type.","Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
+"C:\azure-powershell\src\Package\Debug\ResourceManager\AzureResourceManager\AzureRM.Automation\Microsoft.Azure.Commands.ResourceManager.Automation.dll","Microsoft.Azure.Commands.Automation.Cmdlet.RemoveAzureAutomationHybridWorkerGroup","Remove-AzureRMAutomationHybridWorkerGroup","1","8600","Cmdlet 'Remove-AzureRMAutomationHybridWorkerGroup' has no defined output type.","Add an OutputType attribute that declares the type of the object(s) returned by this cmdlet. If this cmdlet returns no output, please set the output type to 'bool' and make sure to implement the 'PassThru' parameter."
```

Comment:
Should be removed when PassThru is implemented.

Reply:
Fixed.